lxd/container_delete.go

```diff
@@ -71,8 +30,14 @@ func containerDelete(d *Daemon, r *http.Request) Response {
 		return SmartError(err)
 	}
 
+	// TODO: i have added this not sure its a good idea (pcdummy)
```

Comment:
well, lxd owns the path, so nothing else will write in there, which makes it fine to nuke.

What I don't want to end up with is a failed container creation leaving the directory around and then making subsequent container creation of the same name fail, or worse, end up with a mix of the data of both (unpacked on top).